test/countries/test_burundi.py

```diff
@@ -9,84 +9,189 @@
 #  Website: https://github.com/dr-prodigy/python-holidays
 #  License: MIT (see LICENSE file)
 
-import importlib.util
-import unittest
-from datetime import date
+from holidays.countries.burundi import Burundi, BI, BDI
+from test.common import TestCase
 
-import holidays
 
-
-class TestBurundi(unittest.TestCase):
+class TestBurundi(TestCase):
     def setUp(self):
-        self.holidays = holidays.BI()
+        self.holidays = Burundi()
 
-    def test_new_year_day(self):
-        for year in range(1979, 2050):
-            self.assertIn("New Year's Day", self.holidays[date(year, 1, 1)])
+    def test_country_aliases(self):
+        self.assertCountryAliases(Burundi, BI, BDI)
 
-    def test_unity_day(self):
-        self.assertIn("Unity Day", self.holidays[date(2017, 2, 5)])
-        self.assertIn("Unity Day (Observed)", self.holidays[date(2017, 2, 6)])
+    def test_no_holidays(self):
+        self.assertNoHolidays(Burundi(years=1961))
 
-    def test_labour_day(self):
-        self.assertIn("Labour Day", self.holidays[date(2017, 5, 1)])
-        self.assertIn("Labour Day (Observed)", self.holidays[date(2022, 5, 2)])
+    def test_new_year_day(self):
+        for year in range(1962, 2050):
+            self.assertIn("New Year's Day", self.holidays[f"{year}-01-01"])
```

Comment:
Would be useful to have function like `assertHolidaysName`, but with **assertIn** instead of **assertEqual**. @arkid15r, what do you think about it? ;)

Reply:
Hey @KJhellico, I've just filed https://github.com/dr-prodigy/python-holidays/pull/926.

I hope that helps.

Reply:
> I hope that helps.

100%! Thank you! I'll wait for it to be merged.

Reply:
Well, it seemed reasonable to merge it ASAP.
Done.